File: benches/arithmetic.js
const addStage = (startVal) => startVal + 100;
const mulStage = (startVal) => startVal * 2;

const run = (startVal) => {
    return mulStage(addStage(startVal));
};

for (let i = 0; i < 100000; i++) {
    run(10);
}

File: benches/arithmetic.py
def add_stage(start_val):
    return start_val + 100

def mul_stage(start_val):
    return start_val * 2

def run(start_val):
    return mul_stage(add_stage(start_val))

if __name__ == "__main__":
    for i in range(100000):
        run(10)
